openbas-api/src/main/java/io/openbas/rest/tag_rule/form/TagRuleInput.java

```diff
@@ -18,8 +19,10 @@
 public class TagRuleInput {
   @NotBlank(message = MANDATORY_MESSAGE)
   @JsonProperty("tag_name")
+  @Schema(description = "Name of the tag rule")
```

Comment:
Name of the tag rule or name of the tag ? (variable name vs doc is misleading, I think)